### PR TITLE
Feature/twitter into develop 

### PR DESCRIPTION
**Pull request by AkshaySapra**

### CODE REVIEW

**Review by jaypatel-jrvs — CHANGES_REQUESTED** [review]
Remove the target folder from your github repo.
The code itself is good to be merged.

**Review by jaypatel-jrvs — APPROVED** [review]

**Comment by jaypatel-jrvs** [comment]
You can try target/ to ignore the folder. maven has some recommended gitignore files somewhere on the internet or you can look at repos of larger organizations

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            statement.execute();
+            int id = this.getLastVal(CUSTOMER_SEQUENCE);
+            return this.findById(id);
+        }catch(SQLException e){
+            e.printStackTrace();
+            throw new RuntimeException(e);
+        }
+    }
+
+    public List<Customer> findAllSorted(int limit){
```
Comment (andrew-laniak-fuoco) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/CustomerDAO.java:117]: This method name is misleading.

File: core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+
+public class JDBCExecutor {
+
+    public static void main(String... args){
+        DatabaseConnectionManager dcm = new DatabaseConnectionManager("localhost",
```
Comment (andrew-laniak-fuoco) [inline core_java/jdbc/src/main/java/ca/jrvs/apps/jdbc/JDBCExecutor.java:13]: Try using environmental variables.  Hardcoding passwords is dangerous.

**Review by andrew-laniak-fuoco — APPROVED** [review]

**Pull request merged**